js/interface.js

```diff
@@ -151,8 +153,8 @@ function renderDataSources(dataSources) {
   $('#trash-sources').hide();
 }
 
-function renderTrashSources(trashSources) {
-  var html = trashSources.map(function(trashSource) {
+function rendertrashedDataSources(trashedDataSources) {
```

Comment:
use camel case

Reply:
@squallstar this was resolved in previous commits